server/src/main/java/io/druid/server/http/LookupCoordinatorResource.java

```diff
@@ -284,4 +295,346 @@ public Response getSpecificTier(
       return Response.serverError().entity(ServletResourceUtils.sanitizeException(e)).build();
     }
   }
+
+  @GET
+  @Produces({MediaType.APPLICATION_JSON})
+  @Path("/status")
+  public Response getAllLookupsStatus(
+      @QueryParam("detailed") boolean detailed
+  )
+  {
+    try {
+      Map<String, Map<String, LookupExtractorFactoryMapContainer>> configuredLookups = lookupCoordinatorManager.getKnownLookups();
+      if (configuredLookups == null) {
+        return Response.status(Response.Status.NOT_FOUND)
+                       .entity("No lookups found")
```

Comment:
IMHO if we say we produce json we should give a valid json back in case someone is feeding the result into a json parser immediately. (ex: many python libraries)